packages/playground/website/src/github/preview-pr/form.tsx

```diff
@@ -13,16 +13,20 @@ interface PreviewPRFormProps {
 
 const urlParams = new URLSearchParams(window.location.search);
 
+// This structure is from plugin-proxy.php
+// where we set allowed inputs for WordPress and Gutenberg repositories
 export const targetParams = {
 	wordpress: {
 		repo: 'wordpress-develop',
 		workflow: 'Test%20Build%20Processes',
 		artifact: 'wordpress-build-',
+		pull: 'github.com/wordpress/wordpress-develop/pull'
```

Comment:
I tried to think of a more descriptive name, such as `baseUrl`, but there's no `https://` here which makes it more difficult. Let's go with `pull` for now.